packages/webpack-plugin/src/plugin.ts

```diff
@@ -108,6 +109,11 @@ export interface StylableWebpackPluginOptions {
     unsafeMuteDiagnostics?: {
         DUPLICATE_MODULE_NAMESPACE?: boolean;
     };
+    /**
+     * Set the strategy of how to spit the extracted css
+     * This option is only used when cssInjection is set to 'css'
+     */
+    extractMode?: 'single' | 'chunks';
```

Comment:
maybe entries instead of chunks? 
chunks sounds like it might do more than just extract per top level chunk (entry)

Reply:
right